javascript-constructors/student.js

```diff
@@ -0,0 +1,17 @@
+/* exported Student */
+
+const studentProto = {
+  getFullName: function () {
+    return this.firstName + ' ' + this.lastName;
+  },
+  getIntroduction: function () {
+    return 'Hello, my name is ' + this.getFullName() + ' and I am studying ' + this.subject + '.';
+  }
+};
+
+function Student(firstName, lastName, subject) {
+  this.firstName = firstName;
+  this.lastName = lastName;
+  this.subject = subject;
+  Object.setPrototypeOf(this, studentProto);
+}
```

Comment:
Define the methods directly on the `Student` `prototype`, rather than creating a separate prototype object.

Reply:
done! but... the SS is... the same. lol